### ISSUE TITLE
Log scale for x-axis

### ISSUE THREAD

**Issue by GeoffNN** [tensorflow/tensorboard#6085]
Hi!

It would be great to have a way to toggle log scale on the x-axis. See https://github.com/tensorflow/tensorflow/issues/6532
This would allow to check convergence rates (gradient norm vs steps / loss value vs steps when the loss is going to 0), and in my experience is great for debugging.

This feature is available on wandb, it would be great to also have it on tensorboard!

**Comment by rileyajones** [comment]
Which chart and dashboard are you referring to? I'm assuming you are referring to the Time Series dashboards Scalar card?
![image](https://user-images.githubusercontent.com/78179109/206037538-34922a61-2b03-4ea9-a991-2b9c0580bc11.png)

**Comment by nfelt** [comment]
Duplicate of #1147

This seems basically the same request as the longstanding one, #1147 - maybe we can consolidate?

**Comment by GeoffNN** [comment]
Yes, I'm referring to Scalars. Both in the Time series tab and in the Scalars tab.

Apologies @nfelt, I had only seen the older feature request on the TF repo. This is the same request as #1147.
